doc/cascadia/SettingsSchema.md

```diff
@@ -66,7 +66,7 @@ Properties listed below are specific to each unique profile.
 | `suppressApplicationTitle` | Optional | Boolean | `false` | When set to `true`, `tabTitle` overrides the default title of the tab and any title change messages from the application will be suppressed. When set to `false`, `tabTitle` behaves as normal. |
 | `tabTitle` | Optional | String | | If set, will replace the `name` as the title to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application.  |
 | `useAcrylic` | Optional | Boolean | `false` | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. The transparency only applies to focused windows due to OS limitation. |
-| `experimental.retroTerminalEffect` | Optional | Boolean | `false` | When set to `true`, enable retro terminal effects. This is an experimental feature, and its continued existence is not guaranteed. |
+| `experimental.pixelShaderEffect` | Optional | String |  | If set to a file name or preset, compiles the pixel shader effect and applies it the terminal view. Currently 2 presets are supported `RETRO` and `RETROII`. This is an experimental feature, and its continued existence is not guaranteed. |
```

Comment:
So if this is enabled and a user only wants the original retro Terminal effect, will they get it? (I'm not familiar enough with this type of code to figure it out) though  I see the preset 'Retro" is there. Is that the original? Secondly, if a user already has the retroTerminalEffect enabled in their settings.json, you will break them. How will you resolve this? Will you add an alias that states it is depreciated? (recommended)

Reply:
As this was marked as an experimental feature I figured there was less need to retain backwareds compatibility. If it's important keep backwards compatibility for experimental flags then the experimental flag can be reintroduced and code be put into place to check for it.

Reply:
I will add it to `default.json`

Reply:
Yeah it is experimental. But it was released under 1.0 and while we could break people in Preview, we'd need a depreciation path for people on stable. But that also depends on what the maintainers say. If we can break it due to being experimental, then we'd definitely need it defined in defaults.json so users can quickly get it back.

Reply:
I will add support for it them. It's not a big deal to me.

Reply:
We'll have to chat about this on the team. We made the `experimental.` namespace so we could break this _without_ having the preview->stable guarantee, so I'm not certain WSLUser's reasoning holds :smile:

Reply:
"experimental.retroTerminalEffect" is now back.